FirebaseAuth/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
-- [changed] Added a `X-Firebase-GMPID` header to network requests.
+- [changed] Added a `X-Firebase-GMPID` header to network requests. (#9046)
+- [fixed] Added multi-tenancy support to generic OAuth providers. (#9100)
```

Comment:
Recommended practice is to specify the issue number when there is both a bug and PR - #7990 in this case.

Reply:
Ah, gotcha! Done.